lib/wor/authentication/controllers/sessions.rb

```diff
@@ -0,0 +1,120 @@
+module Wor
+  module Authentication
+    module Controllers
+      module Sessions
+        def create
```

Comment:
You could do something like (not sure if this works as it is)
```ruby
def self.included(klass)
  klass.skip_before_action :authenticate_request!, only: [:create]
end
```
This way you guarantee that create method will not require an authenticated user.